src/Apps/Consign/Routes/MarketingLanding/Components/CtaBanner.tsx

```diff
@@ -73,36 +73,9 @@ export const CtaBannerContent = () => {
   }
 
   return (
-    <AppContainer py={[1, 2]}>
-      <HorizontalPadding>
-        <Media at="xs">
-          <Flex>
-            <Button
-              // @ts-ignore
-              as={RouterLink}
-              to="mailto:sell@artsy.net?subject=Inquiry about selling with Artsy"
-              width="100%"
-              onClick={trackGetInTouchClick}
-              variant="primaryWhite"
-              size="small"
-              mr={2}
-            >
-              Get in Touch
-            </Button>
-            <Button
-              // @ts-ignore
-              as={RouterLink}
-              to="/sell/submission/artwork-details"
-              onClick={trackSubmitClick}
-              size="small"
-              width="100%"
-            >
-              Submit an Artwork
-            </Button>
-          </Flex>
-        </Media>
-
-        <Media greaterThanOrEqual="sm">
+    <Media greaterThan="xs">
```

Comment:
How do you feel about using Media here? If not good, let's pair to fix it?

I think it should wrap the CtaBanner component on line 27. I tried doing this, and it breaks the header - when I scroll down and then up again, the content of the header (text and two buttons) disappears. Looks like it is hidden somewhere under the image.
When I use `<FullBleed />` instead of `<FullBleedHeader />` component in the Header.tsx - I don't experience this issue

Reply:
it's okay to use Media here

Reply:
Perfect! Thanks